addons/info/src/components/types/PrettyPropType.js

```diff
@@ -1,8 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-// TODO: FIX this dependency cycle
-/* eslint-disable import/no-cycle */
```

Comment:
ESLint packages update seems quite irrelevant to this PR, and it makes it harder to review =(